miqbox/miqbox.py

```diff
@@ -373,6 +373,30 @@ def start(connection, name):
     except Exception:
         click.echo("Fail to start appliance...")
 
+@cli.command(help="Restart Appliance Server")
+@click.argument("name")
+@connection
+def restart(connection, name):
+    """Stop running appliance
+
+    Args:
+        name: (`str`) appliance name
+    """
+    try:
+        id = int(name)
+        dom = connection.conn.lookupByID(id)
+    except ValueError:
```

Comment:
Are you sure about this error?

Reply:
Yes, ```ValueError``` is occuring when we give name of appliance instead of **ID**. So with this exception we can go with either ```id or name``` of appliance.

Logs:

```
--> 388         id = int(restart)
    389         dom = connection.conn.lookupByID(id)

ipdb> n
ValueError: "invalid literal for int() with base 10: 'n'"
> /home/ghubale/miqbox/miqbox/miqbox.py(388)evmserver()
    387     try:
--> 388         id = int(restart)
    389         dom = connection.conn.lookupByID(id)

ipdb> n
> /home/ghubale/miqbox/miqbox/miqbox.py(390)evmserver()
    389         dom = connection.conn.lookupByID(id)
--> 390     except Exception as e:
    391         print(e)

ipdb> n
> /home/ghubale/miqbox/miqbox/miqbox.py(391)evmserver()
    390     except Exception as e:
--> 391         print(e)
    392         # dom = connection.conn.lookupByName(name)

ipdb> n
invalid literal for int() with base 10: 'n'

```